src/fastoad/models/loops/compute_wing_position.py

```diff
@@ -0,0 +1,58 @@
+"""
```

Comment:
I needed to try a more brutal way to do the job, and I came with the code below.

In the `test_api()` of `test_oad_process.py`, if I deactivate the optimization part to run only the `run_model()`, the execution time of your code on my computer was around 90s. With the code below, when deactivating all solvers except the top ones in `fastoad.toml`, I got the same output results, but in **9 seconds**.

```python
class ComputeWingPosition(om.ExplicitComponent):
    def setup(self):
        self.add_input("data:handling_qualities:static_margin", val=np.nan)
        self.add_input("data:handling_qualities:static_margin:target", val=np.nan)
        self.add_input("data:geometry:wing:MAC:length", val=np.nan, units="m")
        self.add_input("data:weight:aircraft:CG:aft:MAC_position", val=np.nan)
        self.add_input("data:weight:aircraft:CG:aft:x", val=np.nan, units="m")

        self.add_output("data:geometry:wing:MAC:at25percent:x", val=17.0, units="m")

        self.declare_partials(of="*", wrt="*", method="fd")

    def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
        static_margin = inputs["data:handling_qualities:static_margin"]
        target_static_margin = inputs["data:handling_qualities:static_margin:target"]
        l0_wing = inputs["data:geometry:wing:MAC:length"]
        cg_ratio = inputs["data:weight:aircraft:CG:aft:MAC_position"]
        cg_x = inputs["data:weight:aircraft:CG:aft:x"]

        mac_position = (
            cg_x
            + 0.25 * l0_wing
            - cg_ratio * l0_wing
            - (static_margin - target_static_margin) * l0_wing
        )

        outputs["data:geometry:wing:MAC:at25percent:x"] = mac_position
```

Reply:
Thanks! Now working in ea0a276.